code/game/atoms_movable.dm

```diff
@@ -282,6 +282,8 @@
 		A.Bumped(src)
 
 /atom/movable/proc/forceMove(atom/destination)
+	if(!destination)
```

Comment:
This might be a teensy bit faster, which would probably make this check a little nicer given how often this gets called
```suggestion
	if(isnull(destination))
```